### ISSUE TITLE
ZotHero Showing Attachments as Entrys

### ISSUE THREAD

**Issue by BlackF1re0** [deanishe/zothero#29]
Hi,

Just wanted to point out an error/bug which I keep getting after a fresh installation on my macbook pro. So basically it keeps showing me my zotero attachments as "entrys" in my zot Search. Anyone has an idea on how to fix this? I tried reinstalling zotero/alfred multiple times already.

![image](https://user-images.githubusercontent.com/58980856/70997149-a749ab00-20d4-11ea-906a-e87e863bec1d.png)

**Comment by deanishe** [comment]
This *shouldn't* happen. Attachment names should be searched, but the attachments shouldn't actually show up in the results.

Have you tried `zotconf > Reload Zoreto Data`? (Probably won't make any difference.)

I can't replicate this behaviour myself, which makes it almost impossible to figure out what's going on. Is there any chance you could make a new, minimal library that shows the problem and share it with us?

**Comment by BlackF1re0** [comment]
Yeah I agree. To give you some context: Before all of this happened zotero and ZotHero worked perfectly fine, but after a fresh Install of Catalina on my MB Pro 13 (2015), and obviously a fresh download of my Zotero Library through their Zotero Cloud Service this bug started to Appear.

I tried zotconf multiple times, and even did try to change some settings in Zotero but nothing worked! 

So I tried to create as you said @deanishe a new Folder just to try out.
As you see in the picture this new "Folder" did not get downloaded from Zotero Cloud and installed through there. But it did not work either, and the attachment still shows up as an entry.
![Screenshot 2019-12-18 at 14 22 11](https://user-images.githubusercontent.com/58980856/71089884-06292600-21a2-11ea-96d6-b79963c641e3.png)
![Screenshot 2019-12-18 at 14 20 37](https://user-images.githubusercontent.com/58980856/71089885-06292600-21a2-11ea-985f-c01af11cf794.png)

**Comment by deanishe** [comment]
Can you share the library with me (perhaps export it and email it to me), so I can see the problem for myself and perhaps figure out what's going on?

**Comment by BlackF1re0** [comment]
> Can you share the library with me (perhaps export it and email it to me), so I can see the problem for myself and perhaps figure out what's going on?

Were you able to find something?

**Comment by zlf0625** [comment]
Hi, after I refresh the cache (after I installed ZotFile), I found the same problem. I guess it is possible to be caused by storing attachments in another folder outside the Zotero directory with ZotFile?

**Comment by BlackF1re0** [comment]
Hope someone finds a fix for this soon!

**Comment by PhilHem** [comment]
I have the same problem in
Zotero version 5.0.96.3
Zothero 1.3.0
OS X 12.0.1

with a new library 1 one entry.

**Comment by PhilHem** [comment]
I could verify the fix for this issue by using https://github.com/ai0/zothero/commit/331e9e5522940684473309a6aed92d6556fd014b
